src/main/java/carpet/script/utils/ShapeDispatcher.java

```diff
@@ -249,6 +261,8 @@ public static ExpiringShape fromTag(CompoundTag tag)
             put("sphere", creator(Sphere::new));
             put("cylinder", creator(Cylinder::new));
             put("label", creator(DisplayedText::new));
+            put("poly",creator(Polyface::new));
+            put("blockoritemdisplayer",creator(DisplayedItem::new));
```

Comment:
Have you added this to the docs?

Reply:
not yet. i only write the polygon`s doc. 
block or item displayer`s one is not written yet.

Reply:
@Ghoulboy78 one of the reasons is i dont know if those names of arguments are nice for English speakers.

forexample, i dont think "blockoritemdisplayer" is a good name. but i cannot find a better one.
i want to get some feed back and decide those name first.

Reply:
true that.

Reply:
"fix"ed by split it into`'block_displayer'` and `'item_displayer'`: